modules/nf-core/wipertools/fastqgather/main.nf

```diff
@@ -0,0 +1,48 @@
+process WIPERTOOLS_FASTQGATHER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/wipertools:1.1.3--pyhdfd78af_0':
+        'biocontainers/wipertools:1.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fastq_in)
+
+    output:
+    tuple val(meta), path("${prefix}.fastq.gz") , emit: fastq_out
+    path "versions.yml"                         , emit: versions
```

Comment:
```suggestion
    output:
    tuple val(meta), path("${prefix}_gather.fastq.gz") , emit: fastq_out
    path "versions.yml"                         , emit: versions
```

Maybe @SPPearce said something else then feel free to ignore my comment. But if you always add `_gather` to the prefix anyways then I would prefer to see this in the output here directly and also in the meta.yml

Reply:
I don't want gather to always be on the prefix, I want to be able to leave that off.

Reply:
ok because now its always added right above the script block - then I would rather remove it always and leave it to the user to define that in the modules config of their pipeline

Reply:
Hi @famosab, what you suggest was exactly how it was implemented before the @SPPearce review. So I guess he suggested to edit as it is the code now. Here is the point: https://github.com/nf-core/modules/pull/7185#pullrequestreview-2504670480
Am I wrong?

Reply:
You are right :) just ignore these comments then -> and see my comments below